src/theme/components/styledList.ts

```diff
@@ -0,0 +1,25 @@
+import { baseListStyles, baseUnorderedStyles } from "./list";
+
+const StyledList = {
+  baseStyle: ({ textSize = "default" }) => ({
+    fontSize: `text.${textSize}`,
+    ...baseListStyles(false, true),
+    ...baseUnorderedStyles(true),
+  }),
+  variants: {
+    capped: {
+      li: {
+        borderLeft: "1px solid",
+        borderRight: {
```

Comment:
What is this borderRight doing?

Reply:
It wasn't obvious to me but the end border should not display on desktop, only mobile.